models/input/PC_delinted.csv

```diff
@@ -0,0 +1 @@
+../../point_counts/data_ingest/output/PC_delinted.csv
```

Comment:
@matth79 aren't all these .gitignored? Yikes!

Reply:
This commit (7cd9082) that is adding them is also adding exception (!) lines to .gitignore so that they will be unignored.

Whether that is the right approach long-term is still an open discussion in issue #26